express-get-json/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('listening');
+});
```

Comment:
Although this works fine, it is best to put `app.listen` as the last thing in the file. Also you should include the port number in the console log.